PeakLocalisations/Localisation/App Store/Screenshots.xcstrings

```diff
@@ -59,6 +89,12 @@
     "Resolutions.Review" : {
       "extractionState" : "manual",
       "localizations" : {
+        "de" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Blicke auf deinen Fortschritt 2023 zurück"
```

Comment:
I'm not a big fan of using only nouns. Using the verb "zurückblicken" sounds more natural IMHO. I added an `in`, because you're not looking at the progress of the year itself, but at the progress _you_ made _within_ that year.